test/distributed/_composable/fsdp/test_fully_shard_training.py

```diff
@@ -579,16 +557,18 @@ def _test_train_parity_with_activation_checkpointing(
                 torch.manual_seed(iter_idx + 1)  # for dropout determinism
                 losses.append(_model(inp).sum())
                 losses[-1].backward()
-            check_sharded_parity(
-                self, ref_model, model, prefixes_to_ignore=prefixes_to_ignore
-            )
+            if not testing_compile:
```

Comment:
we have loss numeric parity with `torch.compile`. But does not have `.grad` numeric parity

Reply:
How is it that we have loss parity over multiple iterations but not gradient parity? 🤔 

I would expect the losses to diverge on the subsequent iteration then.